packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -414,12 +421,25 @@ export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScr
 
   private _onScroll = () => {
     const { contentContainer } = this;
-
     if (contentContainer) {
       this._stickies.forEach((sticky: Sticky) => {
         sticky.syncScroll(contentContainer);
       });
     }
+    const { stickyTopHeight, stickyAboveClassName } = this.state;
+    if (this.props.onScrollStickyAboveClassName !== undefined && stickyAboveClassName !== undefined && stickyTopHeight > 0) {
```

Comment:
Or simply
if (this.props.onScrollStickyAboveClassName && stickyAboveClassName && stickyTopHeight > 0)

Reply:
Btw, why do we need a check for stickyAboveClassName?